app/templates/components/slide-modal.hbs

```diff
@@ -1,17 +1,12 @@
 <div class = "employee-slide-container" >
-   
         <div class = "leftbar-logo slideleft-button" >
             <div class="logoimg" {{action 'cancelbtn-action'}}>{{#link-to path preventDefault=false }}{{svg-jar 'Cancel' width='20px' height="20px" }}{{/link-to}} </div> 
         </div>
-      
-   
-  <div class = "employee-outerslide"> 
-  </div>
+       
+  <div class = "employee-outerslide"></div>
```

Comment:
For the common components, the class names also can be generic names. As this is not only specific to employee module.